src/main/java/io/github/thebusybiscuit/mobcapturer/listeners/PelletListener.java

```diff
@@ -67,15 +68,24 @@ && canCapture(player, entity)
      */
     @ParametersAreNonnullByDefault
     protected boolean canCapture(Player p, LivingEntity entity) {
-        if (Slimefun.getProtectionManager().hasPermission(p, entity.getLocation(), Interaction.ATTACK_ENTITY)) {
-            if (MobCapturer.getRegistry().getConfig().getBoolean("options.capture-named-mobs")) {
-                return true;
-            } else {
-                return entity.getCustomName() == null;
-            }
-        } else {
+
+        if (!Slimefun.getProtectionManager().hasPermission(p, entity.getLocation(), Interaction.ATTACK_ENTITY)) {
+            //no permission check
+            return false;
+        }
+
+        List<String> ignoredMobNames = MobCapturer.getRegistry().getConfig().getStringList("options.ignored-mobs");
+        if (ignoredMobNames.contains(entity.getCustomName())) {
+            //filter out ignored mob names
             return false;
         }
+
+        if (MobCapturer.getRegistry().getConfig().getBoolean("options.capture-named-mobs")) {
```

Comment:
Any clue how this deals with upper and lower cases. Also does this ignore any formatting like colours and italic/bold?